server/aws-lsp-cloudformation/package.json

```diff
@@ -8,7 +8,7 @@
     },
     "dependencies": {
         "@aws/lsp-core": "^0.0.1",
-        "@aws/aws-lsp-yaml-json": "^0.0.1",
+        "@aws/aws-lsp-json": "^0.0.1",
```

Comment:
This line can cause issues when we bump version, in npm “^0.0.1” can't be resolved to “0.0.2” or “0.1.1". This is because npm considers a major version “the leftmost non zero digit”. So, “0.0.2" is a major version change 
https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004

Reply:
agreed to solve in next PR